projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -788,42 +890,54 @@ function callback( $path = '', $blog_id = 0 ) {
 		$all_formats = get_post_format_strings();
 		$supported   = get_theme_support( 'post-formats' );
 
-		$supported_formats = $response['formats'] = array();
+		$supported_formats = $response['formats'] = array(); //phpcs:ignore
```

Comment:
I believe we'd need to `$response = array()` before this to set it up, which should account for this line and L901 below. I don't recall if this is strictly a PHPCS issue or if PHP will start throwing a notice in a future version for this type of variable assumption.